spec/models/article_spec.rb

```diff
@@ -29,4 +29,39 @@
       end
     end
   end
+
+  context 'scopes' do
+    describe 'visible' do
+      it 'should respond' do
+        expect(Article).to respond_to :visible
+      end
+      it 'should return all articles with visible true' do
+        visible = FactoryGirl.create_list :article, 5, visible: true
+        FactoryGirl.create_list :article, 5, visible: false
+        expect(Article.visible).to match_array visible
+      end
+    end
+
+    describe 'available' do
+      it 'should respond' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.